tasks/interfaces.yml

```
@@ -104,7 +104,7 @@
 
 - meta: flush_handlers
 
-- name: Generate nftables defines
+- name: Generate nftables definitions```
```

Looks like GitHub included the three backticks from my suggestion :/
```suggestion
- name: Generate nftables definitions
```